cryptol-remote-api/check_docs.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd $DIR/docs
+
+export CRYPTOL_SERVER=$(which cryptol-remote-api)
```

Comment:
The other existing RPC script uses a [slightly different method](https://github.com/GaloisInc/cryptol/blob/1d8a2fdb527bc85b0aa7acbf8a0d819a26262184/cryptol-remote-api/run_rpc_tests.sh#L24) to find the server:
```
export CRYPTOL_SERVER=$(cabal v2-exec which cryptol-remote-api)
```
Should these be the same and/or more similar? (I'm just imagining one failing and the other not and that being kind of funny/frustrating.) I think personally I have usually relied on `cabal run` or `cabal exec`, but I certainly don't know what method for finding the executable is the "best".

Reply:
I'll be honest—I cargo-culted much of this script from [`saw-script`'s counterpart](https://github.com/GaloisInc/saw-script/blob/5535b03531d0790cddb01d732be316c96838a8db/saw-remote-api/scripts/check_docs.sh#L9) without thinking too deeply about it. But yes, we should be consistent with the conventions already established in `cryptol`. Fixed.

Reply:
And as luck would have it, that's exactly the change I needed to make anyway in order to fix CI :)